### ISSUE TITLE
Reinventing the scholarly writing stack with the Manubot

### ISSUE THREAD

**Issue by dhimmel** [sparcopen/doathon#21]
## At a glance

* **Submission Name:** Continuous Publishing with the Manubot
* **Contact Lead:** @dhimmel (` Daniel Himmelstein <daniel.himmelstein@gmail.com>`)
* **External Site Link:** https://github.com/greenelab/manubot-rootstock
* **Region**:  Global
* **Issue Area:** OpenAccess, OpenResearch
* **Issue Type:** Project
* **Types of Support Needed:** Coding, GraphicDesign, Users

## Description

[The Manubot](https://github.com/greenelab/manubot-rootstock) let's you write and continuously publish a manuscript on GitHub. Write in markdown and cite using just DOIs or other identifiers. Free yourself from antiquated or proprietary software such as Microsoft Word, LaTeX, EndNote, Mendeley, BibTeX, or MathType. While still young, the Manubot already powers the [Deep Review](https://github.com/greenelab/deep-review) (a massively collaborative review paper), the [Sci-Hub Coverage Study](https://github.com/greenelab/scihub-manuscript), and [has even reproduced](https://steemit.com/manubot/@dhimmel/how-i-used-the-manubot-to-reproduce-the-bitcoin-whitepaper) the Bitcoin Whitepaper.

The Manubot aims to make scholarly publishing transparent, reproducible, and collaborative. It's an open source project and gratis to use. In a world of ever-increasing scholarly publishing costs, **take a stand to show how publishing can be entirely free of charge**, while also offering a superior user experience to existing journals. More details in [these slides](http://slides.com/dhimmel/manubot#/).

## What are we working on during the do-a-thon? What kinds of support do we need?

At the do-a-thon, we will first help users setup new Manubot instance for their manuscripts. The [setup process](https://github.com/greenelab/manubot-rootstock/blob/master/SETUP.md) can be a bit technical, so this tutorial may be helpful for many users.

Next, attendees are encouraged to help improve the Manubot and contribute features they'd like to see. While the Manubot already supports tables, figures, equations, citation by identifiers, custom bibliographic styles, templating, timestamping, and more, there's still work needed to perfect the system and make it more user friendly as well as interoperable with the existing scholarly infrastructure.

Specifically expertise that's relevant for contributions is frontend or Python development, creating bots, documentation writing, continuous integration, git / GitHub, Pandoc, publishing experience, and copyediting. If you have other skills, we can likely use them as well!

##  How can others contribute?

Try setting up your own manuscripts!

**Specific issues for anyone looking for tasks are:**

+ diffing manuscripts: https://github.com/greenelab/manubot-rootstock/issues/54
+ perfecting the bibliographic style: https://github.com/greenelab/manubot-rootstock/issues/53
+ archiving manuscripts and comments on GitHub: https://github.com/greenelab/manubot-rootstock/issues/45
+ measuring manuscript contribution: see https://github.com/greenelab/meta-review/pull/16
+ automating the setup process and further automation in general
+ improving the viewer or integrating with JATS-based viewers: https://github.com/greenelab/manubot-rootstock/pull/82
+ retrieve ISBN citation metadata: https://github.com/greenelab/manubot/issues/14
+ retrieve legal citation metadata: https://github.com/greenelab/manubot/issues/1
+ improving citation parsing: https://github.com/greenelab/manubot/issues/13

_This post is part of the OpenCon 2017 Do-A-Thon. Not sure what's going on? [Head here](http://doathon.opencon2017.org/)._

**Comment by chartgerink** [comment]
Way to get the OpenCon feeling started early @dhimmel! :-)

**Comment by npscience** [comment]
Sign me up as a tester to get started with ManuBot @dhimmel!

**Comment by dhimmel** [comment]
Excited for the do-a-thon tomorrow. Attendees are encouraged to bring a laptop, so you can get involved and setup your own manuscripts. However, even if you don't have a laptop, your feedback will be appreciated!

**Comment by dhimmel** [comment]
One more note: You don't need to know Python or any other programming language to attend. If you write manuscripts / scholarly documents, then **you are the target audience**! You will likely get the most out of Manubot if you're computer-savvy, but we're looking for diverse feedback, so don't be shy.

**Comment by RaoOfPhysics** [comment]
@dhimmel: Do you know where you might be camping? I'll come find you.

**Comment by RaoOfPhysics** [comment]
Added two issues:

- https://github.com/greenelab/manubot-rootstock/issues/85
- https://github.com/greenelab/manubot-rootstock/issues/86

**Comment by dhimmel** [comment]
For new mansucript setup, see https://github.com/greenelab/manubot-rootstock/blob/master/SETUP.md.

**Comment by broadwym** [comment]
https://broadwym.github.io/manu1/ (editing manuscript in-progress)

**Comment by schliebs** [comment]
Here is a link to the instances: https://github.com/schliebs/open_manuscript

**Comment by npscience** [comment]
Sorry I didn't get to this today! Genuinely want to know how this works / try it out, for obvious reasons. Is it possible to do this remotely another time please?

Thanks!!

**Comment by dhimmel** [comment]
> Is it possible to do this remotely another time please?

Certainly! It's always possible, and we're trying to document the setup process, so most users won't need any in-person help. At the moment, things are a bit more complicated for Windows users... but macOS and Linux users should have an easier time. @npscience also happy to help you out or discuss me the project more generally. Just drop me an email anytime.

**Comment by dhimmel** [comment]
# 2018 OpenCon Update

I won't be at OpenCon 2018 sadly, but attendees should feel free to contribute or experiment with Manubot as part of the Do-A-Thon.

For some more up-to-date information on the project, see

+ the project manuscript titled [Open collaborative writing with Manubot](https://greenelab.github.io/meta-review/)
+ [slides](https://slides.com/dhimmel/force2018) from FORCE2018

Some things Do-A-Thon attendees could do in 2018 (or future years):

+ Set up a Manubot manuscript for their next paper (or migrate an existing paper to Manubot). We are especially interested in suggestions or pain points users experience.
+ Look through open GitHub Issues and implement a solution or contribute a discussion. It may be a good idea to comment here if you're interested with your skillset and I can point you in the right direction.

Happy DO-ing!
